package.json

```diff
@@ -34,7 +34,7 @@
   "dependencies": {
     "@rollup/plugin-buble": "^0.21.3",
     "autoprefixer": "^7.1.2",
-    "bootstrap": "^4.0.0",
+    "bootstrap": "^4.6.1",
```

Comment:
Assume this doesn't have Relish effects?

Reply:
We were actually running `4.6.0` thanks to the `^`... I could probably change this back to `^4.0.0` ?